resources/lang/en_US/validation.php

```diff
@@ -46,6 +46,7 @@
     'reconciled_forbidden_field'      => 'This transaction is already reconciled, you cannot change the ":field"',
     'deleted_user'                    => 'Due to security constraints, you cannot register using this email address.',
     'rule_trigger_value'              => 'This value is invalid for the selected trigger.',
+    'rule_action_expression'          => 'Invalid expression. :error',
```

Comment:
where are the translation strings for the syntax errors?

Reply:
These are unfortunately hard-coded in the expression parser's [`SyntaxError` class](https://github.com/symfony/expression-language/blob/7.0/SyntaxError.php) and within the lexer. I doubt there will be many changes to the format of these error messages, so it should be possible to parse the data out of them and then create translatable messages.